src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -39,7 +42,19 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
 
   private browserBehavior: DefaultBrowserBehavior = new DefaultBrowserBehavior();
 
-  constructor(private logger: Logger, private meetingSession: MeetingSession) {}
+  constructor(
+    private logger: Logger,
+    private meetingSession: MeetingSession,
+    configuration?: MeetingReadinessCheckerConfiguration
+  ) {
+    this.configuration = new MeetingReadinessCheckerConfiguration();
```

Comment:
As in `MeetingSessionConfiguration`, how about defining default values in the `MeetingReadinessCheckerConfiguration` class itself?

The constructor can be simplifed into: 
```
constructor(
    private logger: Logger,
    private meetingSession: MeetingSession,
    private configuration: MeetingReadinessCheckerConfiguration = new MeetingReadinessCheckerConfiguration()
) {}
```

Reply:
Nice suggestions. I have updated the code.